src/main/java/core/basesyntax/service/impl/DataValidationImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.DataValidation;
+
+public class DataValidationImpl implements DataValidation {
+    private static final String VALID_REGEX = "[bspr],[a-z]+,[0-9]+";
+
+    @Override
+    public boolean checkLine(String line) {
+        return line.matches(VALID_REGEX);
```

Comment:
if input is broken it's better to throw exception instead of skipping the line

Reply:
Done.